### PR TITLE
feat: include systemd service file in repo

### PR DESCRIPTION
**Pull request by rbjorklin**
Include systemd service file in repo to allow templating.

### CODE REVIEW

File: nomad/defaults.yaml [hunk 1]
```diff
@@ -4,9 +4,8 @@
 
 nomad:
   bin_dir: '/usr/bin'
-  config_dir: '/etc/nomad'
-  version: '0.8.7'
-  service_hash: fdc512a9e7e6a55303f12d4405676ec7c8116af6
+  config_dir: '/etc/nomad.d'
```
Comment (vutny) [inline nomad/defaults.yaml]: I think you'd need to update the `pillar.example` file as well.

**Review by vutny — CHANGES_REQUESTED** [review]
I haven't used this formula and Nomad in particular, but the changes look good to me.
One last thing we need to address is fixing paths in `test_nomad_config.py` for CI to pass successfully.

**Comment by rbjorklin** [comment]
It seems Travis fails to verify that the service is in fact running. Any pointers on how to read the output from `systemd status nomad` within the tests would be appreciated.

File: pillar.example [hunk 2]
```diff
@@ -2,39 +2,27 @@
...
+                {%- endfor %}"
     server:
       enabled: True
       bootstrap_expect: 1
       encrypt: "AaABbB+CcCdDdEeeFFfggG=="
+      server_join:
+        retry_join:
+        {% for _, addrs in salt.saltutil.runner('mine.get', tgt='*', fun='network.ip_addrs') | dictsort() %}
+          - {{ addrs[0] }}
+        {% endfor %}
```
Comment (vutny) [inline pillar.example]: @rbjorklin I think this addition is preventing Nomad agent to start.

The function must be allowed to call from Salt "mine" explicitly, but the tests seems do not do that.
So, when there are nothing rendered within the for loop, the `retry_join` value would be `None`, which is invalid I suppose.
I suggest you to check if there any values come out from the `mine.get` call first, and only then iterate over them to render complete `server_join` dictionary.

**Comment by rbjorklin** [comment]
Long abandoned. You should be using Kubernetes.

**Pull request closed without merging**